esp.d.ts

```diff
@@ -31,13 +31,13 @@ export class Router {
     runAction<T>(modelId:string, action:(model : T) => void)  : void;
     getEventObservable<T, TEvent, TContext>(modelId:string, eventType:string, observationStage?: string) : EventObservable<T, TEvent, TContext>;
     getModelObservable<T>(modelId:string) : Observable<T>;
+    observeEventsOn(modelId: string, object : any, methodPrefix?: string) : Disposable;

```

Comment:
it will have no effect as you'd never actually provide `TModel` and the compiler would infer it. I guess you could provide it if you wanted to be extract strict so not a problem to add it